### PR TITLE
464 implement map in resultroutes

### PR DESCRIPTION
**Pull request by dkrakau**
### Description

Issue:
#464-implement-map-in-resultroutes 

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [ ] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [ ] No tests required
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by Escowei98** [comment]
Cant see the whole Routedetails (cant scroll down)

File: frontend/src/components/ResultRoutes/ResultRoutes.tsx [hunk 1]
```diff
@@ -20,29 +20,101 @@ import "swiper/css/pagination";
...
   const [slideName, setSlideName] = useState<string>("Verfügbare Routen");
   const [activeSlideIndex, setActiveSlideIndex] = useState<number>(0);
+
   const [selectedJourney, setSelectedJourney] = useState<IJourney | null>(null);
+  const [activeJourneyIndex] = useState<number>(0);
+
   const [swiper, setSwiper] = useState<Swiper | null>(null);
 
+  const result = useJourneyQuery(origin, destination, new Date(departureTime), false);
+  const iJourneys = result.type === "success"
```
Comment (Escowei98) [inline frontend/src/components/ResultRoutes/ResultRoutes.tsx:75]: use TripOptionsDisplay Component

File: frontend/src/components/ResultRoutes/ResultRoutes.tsx [hunk 2]
```diff
@@ -61,11 +133,18 @@ const ResultRoutes: React.FC = () => {
   };
 
   useEffect(() => {
+    console.log("ResultRoutes");
```
Comment (Escowei98) [inline frontend/src/components/ResultRoutes/ResultRoutes.tsx:136]: remove console.logs

File: frontend/src/components/map/MapMarker.tsx [hunk 3]
```diff
@@ -141,7 +121,9 @@ export const OriginMapMarker: React.FC<{
 function getOriginIcon(type: string): string {
   switch (type) {
```
Comment (Escowei98) [inline frontend/src/components/map/MapMarker.tsx:122]: remove switch type if all types return same icon

File: frontend/src/components/ResultRoutes/ResultRoutes.tsx (hunk 2 above)
Comment (dkrakau) [reply frontend/src/components/ResultRoutes/ResultRoutes.tsx]: Is removed.

File: frontend/src/components/map/MapMarker.tsx (hunk 3 above)
Comment (dkrakau) [reply frontend/src/components/map/MapMarker.tsx:92]: Empty types are removed.

**Review by Escowei98 — APPROVED** [review]

**Pull request merged**